pkg/operator/ceph/version/version.go

```diff
@@ -0,0 +1,100 @@
+package version
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+
+	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
+)
+
+type CephVersion struct {
+	Major int
+	Minor int
+	Extra int
```

Comment:
How about `Patch`?

Reply:
I initially used `Patch`, and then noticed that Ceph used `patch` in one place, but `extra` nearly everywhere.

Reply:
Interesting, I always thought of it as the patch. Sounds good to use `Extra` if that's what ceph is calling it.

Reply:
yeh, i think patch is more common (especially since it is used by semantic versioning)